refinedstorage2-platform-fabric/src/main/java/com/refinedmods/refinedstorage2/platform/fabric/integration/recipemod/rei/FilteredResourceFocusedStackProvider.java

```diff
@@ -0,0 +1,38 @@
+package com.refinedmods.refinedstorage2.platform.fabric.integration.recipemod.rei;
+
+import com.refinedmods.refinedstorage2.platform.api.integration.recipemod.IngredientConverter;
+import com.refinedmods.refinedstorage2.platform.api.resource.filter.FilteredResource;
+import com.refinedmods.refinedstorage2.platform.common.containermenu.AbstractResourceFilterContainerMenu;
+import com.refinedmods.refinedstorage2.platform.common.screen.AbstractBaseScreen;
+
+import dev.architectury.event.CompoundEventResult;
+import me.shedaniel.math.Point;
+import me.shedaniel.rei.api.client.registry.screen.FocusedStackProvider;
+import me.shedaniel.rei.api.common.entry.EntryStack;
+import net.minecraft.client.gui.screens.Screen;
+
+public class FilteredResourceFocusedStackProvider implements FocusedStackProvider {
+    private final IngredientConverter converter;
+
+    public FilteredResourceFocusedStackProvider(final IngredientConverter converter) {
+        this.converter = converter;
+    }
+
+    @Override
+    public CompoundEventResult<EntryStack<?>> provide(final Screen screen, final Point mouse) {
+        if (screen instanceof AbstractBaseScreen<?> containerScreen
+            && containerScreen.getMenu() instanceof AbstractResourceFilterContainerMenu
```

Comment:
yeah second part is not needed, first one is thought, since the function on the next line is only on AbstractBaseScreen. If I remember correctly it was needed in an earlier version of this PR.